.github/workflows/pr.yml

```diff
@@ -0,0 +1,98 @@
+name: PR Checks
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+    
+jobs:
+  swiftlint:
+
```

Comment:
Could you add `name: Run SwiftLint` here so that it displays nicely in the workflow details?